draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2116,7 +2116,281 @@ uCDN-A                         dCDN-B                         dCDN-C
+		Trigger Status and Error.v2 Description objects.
+	</t>
+	<t>
+		The following table defines the initial ObjectListType JSON string values
+	</t>
+
+	<texttable style="full">
+		<ttcol align="left"> JSON string</ttcol>
+		<ttcol align="left"> Description</ttcol>
+		<ttcol align="left"> Specification</ttcol>
+		<ttcol align="left"> Protocol Specification</ttcol>
+
+		<c>hls</c>
+		<c>HTTP Live Streaming</c>
+		<c>RFCthis</c>
+		<c><xref target="RFC8216">RFC 8216</xref></c>
+
+		<c>mss</c>
+		<c>Microsoft Smooth Streaming</c>
+		<c>RFCthis</c>
+		<c><xref target="MSS">MSS</xref></c>
+
+		<c>dash</c>
+		<c>Dynamic Adaptive Streaming over HTTP (MPEG-DASH)</c>
+		<c>RFCthis</c>
+		<c><xref target="MPEG-DASH">MPEG-DASH</xref></c>
+
+		<c>json</c>
+		<c>JSON serialized object list</c>
+		<c>RFCthis</c>
+		<c><xref target="object-list-type-json">JSON</xref></c>
+
+		<c>text</c>
+		<c>Object list in text format</c>
+		<c>RFCthis</c>
+		<c><xref target="object-list-type-text">Text</xref></c>
+        </texttable>
+	<t>[RFC Editor: Please replace RFCthis with the published RFC number for this document.]</t>
+
+	<section title="JSON Serialized Object List" anchor="object-list-type-json">
+	<t>
+		This ObjectList type specifies a collection of objects encoded in JSON format, each entry being encoded as an <xref target="object-entry"> ObjectEntry</xref> object.  The entries in the JSON object list MAY have object list type specified, allowing for a recursive object list structure.
+	</t>
+	</section>
+
+	<section title="Text Object List" anchor="object-list-type-text">
+		<t>
+			The text object list type specifies a collection of files in text encoding, each line specifying object URI. Unlike the <xref target="object-list-type-json">JSON</xref> object list, the text-based object list will not support a recursive object list structure, and every object specified in it SHOULD be downloaded without additional processing.
+		</t>
+		</section>
+	</section>
+
+	<section title="ObjectEntry" anchor="object-entry">
+	<t>
+		ObjectEntry is a metadata object describing an object and associated metadata, to be used in JSON-encoded <xref target="object-list">ObjectList</xref> objects.
+	</t>
+	<t>
+		Following is an example of JSON serialized ObjectEntry objects
+		<artwork><![CDATA[
+[
+  {“href”:“https://example.com/hls/35cdc008/index.m3u8”,“type":“hls"},
+  {“href”:“https://example.com/dash/35cdc008/main.mpd”,“type":“dash"},
+  {“href”:“https://example.com/img/35cdc008/thumb-l.jpg”,“size”:10260},
+  {“href”:“https://example.com/img/35cdc008/thumb-s.jpg”,“size”:1453}
+]
+
+		]]></artwork>
+	</t>
+
+	<t>The ObjectEntry properties are as follows:</t>
+	<t>
+		<list style="empty" hangIndent="3">
+		<t>
+			<list style="hanging" hangIndent="3">
+				<t hangText="Name: href">
+        			<vspace blankLines="1"/>
+				Description: Object URL
+				<vspace blankLines="1"/>
+				Value: String in URL format
```

Comment:
Maybe: "A URL represented as a JSON string." (This is consistent with other parts of the document.)